src/playback/view/internal/outputresourceitem.cpp

```diff
@@ -8,7 +8,12 @@
 using namespace mu::playback;
 using namespace mu::audio;
 
-static const QString NO_FX_MENU_ITEM_ID = QString::fromStdString(mu::trc("playback", "No effect"));
+static const QString& NO_FX_MENU_ITEM_ID()
+{
+    static std::string id = mu::trc("playback", "No effect");
```

Comment:
I think this still won't work because it's still a static string. So it will get the correct value when it is called for the first time, but it won't be updated until the app is restarted when the user changes the app language.

Also, I think it is better to use `mu::qtrc` directly, because `mu::trc` calls `QCoreApplication::translate()` just like `mu::qtrc` and then converts it to a std::string, and here the std::string is converted back to a QString. So, using `mu::qtrc` here would eliminate two conversions.